types/react-native/index.d.ts

```diff
@@ -6885,6 +6885,11 @@ export class ScrollView extends ScrollViewBase {
      */
     scrollToEnd(options?: { animated: boolean }): void;
 
+    /**
+     * Displays the scroll indicators momentarily.
+     */
+    flashScrollIndicators(): void;
```

Comment:
Shall we add a test for this?

Reply:
I think this is an example of what is fine not to test, it won’t add anything. I think tests are important to test the types that TS will enforce through the system, but this method takes no arguments nor does it return anything.

Reply:
Good point, thanks for the clarification!